pkg/platform/abstract/platform.go

```diff
@@ -279,6 +279,27 @@ func (ap *Platform) EnrichLabels(ctx context.Context, labels map[string]string)
 	}
 }
 
+func (ap *Platform) EnrichExplicitAckParams(ctx context.Context, functionConfig *functionconfig.Config) error {
```

Comment:
move below `enrichTriggers` function.
also, why is it a public method? if located on abstract, then both kube and local will use it, no need to expose it for them

Reply:
Correcto